packages/plugins/camshot/src/lib/commands/recover-camshot.command.ts

```
@@ -0,0 +1,8 @@
+import { ID } from '@gauzy/contracts';
+
+import { ICommand } from '@nestjs/cqrs';
+import { ID } from '@gauzy/contracts';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ID'. Consider to delete it or rename it.

'ID' is defined here:


(lint/suspicious/noRedeclare)